src/timely-util/src/builder_async.rs

```diff
@@ -496,9 +538,8 @@ impl<G: Scope> OperatorBuilder<G> {
                     // also register all the handles for drainage
```

Comment:
This comment is now outdated, there is no more registering of handles for drainage.